### PR TITLE
Add additional steps to input.py (only for internal-use branch)

### PR DESCRIPTION
**Pull request by mmasciov**
#### PR description:

Add initialStepPreSplitting and pixelPairStep to input.py.
Additionally, add "all" option for 'timing' to enable both summary and DQM-by-path.

### CODE REVIEW

File: RecoTracker/MkFit/python/input.py [hunk 1]
```diff
@@ -90,10 +90,13 @@ def apply(process, options):
             process.options.wantSummary = True
         elif options.timing == "FastTimerService":
             process.FastTimerService.enableDQMbyPath = True
+        elif options.timing == "all":
+            process.options.wantSummary = True
+            process.FastTimerService.enableDQMbyPath = True
```
Comment (slava77) [inline RecoTracker/MkFit/python/input.py:95]: does this actually work?
I recall that Timing service and FastTimerService have some destructive interference (they run, but the timing results are wrong)

Comment (mmasciov) [reply RecoTracker/MkFit/python/input.py:95]: It runs, but I never really used it for a measurement/comparison. 
If there's any doubt about it, I'll just remove this option.

Comment (makortel) [reply RecoTracker/MkFit/python/input.py:95]: I'd suggest to drop it because of it messing up the results.

Comment (mmasciov) [reply RecoTracker/MkFit/python/input.py]: Removed it.

**Pull request merged**